.github/workflows/fuzz.yml

```diff
@@ -0,0 +1,47 @@
+name: Fuzz
+on:
+  schedule:
+    # Once a week
+    - cron: '0 0 * * 0'
```

Comment:
I'm not sure if we need to be prodded to fix this so often. The fuzzing harness tends to be used much less frequently than once every week, so I'd prefer if it was fixed whenever someone wants to use it.

(at least I think this makes more sense until there's good fuzzers that work on stable Rust)

Reply:
I'm happy to change this to once a month or some other period. Having said that, with this configuration you shouldn't be prodded once a week. More so if it breaks (doesn't build) again you'll be prodded at the end of the week. Otherwise, the workflow should silently succeed.

Reply:
Right. I'm not sure if that's good value – we don't need to get this fixed within a week if it starts failing, or at all really, until someone wants to run the fuzz tests again.

Reply:
True, on the other hand, there's no reason not to fix it quickly either, and any breakage should be pretty easy to handle. And I'm not sure who gets prodded anyway, I think I only get notifications for my own PRs.

Reply:
If this fuzz test isn't really used should I instead just delete it? The motivation for including the workflow job was to prevent bitrot. A broken fuzz harness serves no purpose whatsoever. If we are just looking at this from the perspective of bitrot, there are two angles;
1. This fuzz test doesn't add value and should be deleted.
2. This fuzz test has some limited value and should be kept up to date.

I don't feel strongly about either of those two options. But it does feel a little strange to merge a commit fixing this with the possibility that it could silently break again at any time.

Reply:
I say let's give it a try. If it becomes annoying, we can disable or make it run less often.